pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -318,6 +318,11 @@ func (policy *regoEnforcer) getReasonNotAllowed(enforcementPoint string, input i
 
 func (policy *regoEnforcer) redactSensitiveData(input inputData) inputData {
 	if v, k := input["envList"]; k {
```

Comment:
unrelated, should this be `v, ok`?

Reply:
yes. shall i do another PR for that?

Reply:
should it be `ok` or `exists` or what? i cant remember the go convention.